resinhup.py

```diff
@@ -111,17 +112,26 @@ def main():
         if not setConfigurationItem(args.conf, "config.json", "type", "production"):
             return False
 
+    # Handle old boot partitions
+    r = Repartitioner(args.conf)
+    if not r.increaseResinBootTo(40):
```

Comment:
Yes - out current boot partition is 40 MiB. I didn't tested on a updated device but should not run on them. Will test when have a chance.